cmd/syncer/main.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The KCP Authors.
+Copyright 2022 The KCP Authors.
```

Comment:
Please don't change copyright dates of existing files (or if you're going to, I'd say only make additive changes; e.g. 2021-2022)